frontend/src/components/BreadcrumbNav.jsx

```diff
@@ -10,83 +10,115 @@ import {
+                    id="SeriesBreadCrumb"
+                    className="SeriesBreadCrumb"
+                    aria-label={'series: ' + selectedBranchState.name}
+                >
+                    {selectedBranchState.name}
+                </StyledLink>
+            </li>
+        </React.Fragment>
     );
 };
 
-const BreadcrumbItemBuild = () => {
+const BreadcrumbItemBuild = props => {
     const { buildId, seriesId } = useParams();
     return (
-        <StyledInnerDiv>
-            <BreadcrumbItemSeries /> <span>/</span>
-            <StyledLink
-                to={`/series/${seriesId}/build/${buildId}/overview`}
-                id="BuildBreadCrumb"
-                className="BuildBreadCrumb"
-            >
-                {buildId}
-            </StyledLink>
-        </StyledInnerDiv>
+        <React.Fragment>
+            <BreadcrumbItemSeries />
+            <li>
+                <span>/</span>
+                <StyledLink
+                    aria-current={props.current ? 'page' : false}
+                    to={`/series/${seriesId}/build/${buildId}/overview`}
+                    id="BuildBreadCrumb"
+                    className="BuildBreadCrumb"
+                    aria-label={`buildId ${buildId}`}
+                >
+                    {buildId}
+                </StyledLink>
+            </li>
+        </React.Fragment>
     );
 };
 
-const BreadcrumbItemSuite = () => {
-    const { suiteId } = useParams();
+const BreadcrumbItemSuite = props => {
+    const { seriesId, buildId, suiteId } = useParams();
     return (
-        <StyledInnerDiv>
-            <BreadcrumbItemBuild /> <span>/</span>
-            <span id="SuiteBreadCrumb">{suiteId}</span>
-        </StyledInnerDiv>
+        <React.Fragment>
+            <BreadcrumbItemBuild />
+            <li>
+                <span>/</span>
+                <StyledLink
+                    aria-current={props.current ? 'page' : false}
+                    to={`/series/${seriesId}/build/${buildId}/suite/${suiteId}/history`}
+                    id="SuiteBreadCrumb"
+                    className="SuiteBreadCrumb"
+                    aria-label={`SuiteId ${suiteId}`}
+                >
+                    {suiteId}
+                </StyledLink>
+            </li>
+        </React.Fragment>
     );
 };
 
 const BREADCRUMB_STATUS = {
-    team: <BreadcrumbItem />,
-    series: <BreadcrumbItemSeries />,
-    build: <BreadcrumbItemBuild />,
-    suite: <BreadcrumbItemSuite />,
+    team: <BreadcrumbItemTeam current />,
+    series: <BreadcrumbItemSeries current />,
+    build: <BreadcrumbItemBuild current />,
+    suite: <BreadcrumbItemSuite current />,
 };
 
 const BreadcrumbNav = ({ status }) => {
     return (
-        <BreadcrumbContainer id="breadCrumbNav" status={status}>
-            <StyledInnerDiv>{BREADCRUMB_STATUS[`${status}`]}</StyledInnerDiv>
+        <BreadcrumbContainer id={'breadCrumbNav'} status={status}>
+            <nav aria-label="Breadcrumb">
```

Comment:
I don't know what the policy is, but since Epimetheus has localization, it would be nice to use it also with aria-label.s

Reply:
This will also have to be done in future with more knowledge about localization if that's be okay